src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/locationsinsideprison/LocationsInsidePrisonClient.kt

```diff
@@ -36,6 +36,7 @@ class LocationsInsidePrisonClient(private val locationsInsidePrisonApiWebClient:
     .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
     .block()
 
+  @Cacheable(CacheConfiguration.LOCATION_BY_KEY_CACHE_NAME)
```

Comment:
Added short term cache, this will be called multiple times on the amend endpoint.